repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkCommandLine.scala

```diff
@@ -23,10 +23,15 @@ import scala.Predef._
 /**
  * Command class enabling Spark-specific command line options (provided by
  * <i>org.apache.spark.repl.SparkRunnerSettings</i>).
+ *
+ * @example new SparkCommandLine(Nil).settings
+ *
+ * @param args The list of command line arguments
+ * @param settings The underlying settings to associate with this set of
+ *                 command-line options
  */
 class SparkCommandLine(args: List[String], override val settings: Settings)
```

Comment:
Oh, do you use the DeveloperApi for entire classes? Or are you saying that since the purpose I described for _SparkCommandLine_ was to retrieve settings?

I didn't mark _SparkILoop_, _SparkIMain_, _SparkHelper_ (forced to be public due to packaging), or _SparkCommandLine_ as DeveloperApi on the class level. I can go back and do that if that's the way things are normally done.

Or, I can just add it to SparkCommandLine.